configs/ot2_defconfig

```diff
@@ -137,7 +137,6 @@ BR2_PACKAGE_WPA_SUPPLICANT_WPA_CLIENT_SO=y
 BR2_PACKAGE_WPA_SUPPLICANT_PASSPHRASE=y
 BR2_PACKAGE_WPA_SUPPLICANT_DBUS=y
 BR2_PACKAGE_WPA_SUPPLICANT_DBUS_INTROSPECTION=y
-BR2_PACKAGE_FILE=y
```

Comment:
No objection from me, but is this removal of the `file` program intentional?

Reply:
yup - no space

Reply:
😬